powerdns/models/powerdns.py

```diff
@@ -390,6 +390,53 @@ class Meta:
         verbose_name = _("record")
         verbose_name_plural = _("records")
 
+    def request_factory(operation):
```

Comment:
this looks like class-level method, but it's regular function, isn't it? if so, then why it's defined in this class? :)